yourcfp/users/decorators.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth import REDIRECT_FIELD_NAME
+from django.contrib.auth.decorators import user_passes_test
+
+
+def organizer_required(function=None, redirect_field_name=REDIRECT_FIELD_NAME, login_url='login'):
+    '''
+    Decorator for views that checks that the logged in user is a organizer,
+    redirects to the log-in page if necessary.
+    '''
+    actual_decorator = user_passes_test(
+        lambda u: u.is_active and u.is_organizer,
+        login_url=login_url,
+        redirect_field_name=redirect_field_name
+    )
+    if function:
+        return actual_decorator(function)
+    return actual_decorator
+
+
```

Comment:
Login decorator would be per

- admin
- organiser (reviewers, sponsors, volunteers)
- speaker